src/providers/vk.ts

```diff
@@ -0,0 +1,54 @@
+import type { OAuthConfig, OAuthUserConfig } from "."
+
+export interface VkProfile {
+  response: Array<{
+    id: number
+    first_name: string
+    last_name: string
+    photo_100: string
+    can_access_closed: boolean
+    is_closed: boolean
+  }>
```

Comment:
```suggestion
  /** [Additional Fields 1](https://vk.com/dev/fields) | [Additional Fields 2](https://vk.com/dev/fields_2) */
  response: Array<{
    id: number
    first_name: string
    last_name: string
    photo_100: string
    can_access_closed: boolean
    is_closed: boolean
  }>
```

I found a bunch of extra fields, let's add it as a JSDoc.

Feel free to add all of those to the interface as well, for easier discovery for the user.